setup.py

```diff
@@ -290,7 +293,7 @@
             ],
             "include_dirs": _cmplr_include + _mathlib_include + _project_backend_dir + _dpctrl_include,
             "library_dirs": _mathlib_path + _omp_libpath + _dpctrl_libpath,
-            "runtime_library_dirs": _project_rpath + _mathlib_rpath + _cmplr_rpath + _omp_rpath + _dpctrl_libpath,
+            "runtime_library_dirs": _runtime_library_dirs,
```

Comment:
We can remove all these runtime library paths for all configurations

Reply:
done